src/json/members/vavali.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Vedha Avali",
+  "pennkey": "vavali",
+  "major": "Computer Science",
+  "school": "Engineering",
+  "bio": "Hi, I'm Vedha and my favorite things are food and my golden retriever.",
+  "hometown": "Pittsburgh, PA",
+  "team": "Penn Mobile",
+  "roles": ["Android Mobile Engineer"],
+  "photo": "https://imgur.com/bVCbRVf",
```

Comment:
I think you copied the wrong photo URL. Try right clicking the image itself and selecting "copy image address".

Reply:
Just fixed it! Thanks